7-high-level-structural-patterns.rst

```diff
@@ -1172,6 +1172,23 @@ Examples
 Images
 ******
 
+#.	:html:`<img>` Each image has a unique :html:`id` attribute.
+
+     #.  That attribute's name is :value:`illustration-` followed by :value:`-N`, where :value:`N` is the sequence number of the element starting at :value:`1`.
+
+     #.  If the image is inline with the text, the :html:`id` is on the :html:`<img>` element.
+
+ 			.. code:: html
+
+ 				<img alt="..." src="..." id="illustration-1" />
+
+ 	#.	When contained in a :html:`<figure>` element, the :html:`<img>` element does not have an :html:`id` attribute; instead the :html:`<figure>` element has the :html:`id` attribute.
```

Comment:
This looks like copy and paste of https://standardebooks.org/manual/1.4.0/single-page#7.8.4.1 but that line is still below?